functests/1_performance/performance.go

```diff
@@ -135,6 +136,17 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			}
 		})
 
+		It("[test_id:32375][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] initramfs should not have injected configuration", func() {
```

Comment:
this pops out after a rebase ? try rebasing anyway

Reply:
may be because he has second commit in this PR?
https://github.com/openshift-kni/performance-addon-operators/pull/305/commits/a617ae386c529748702682d59d0a859d2ee30568

Reply:
Bad rebase for sure